src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/CoreWindowExtension.Mouse.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
What's the source of this code? Might be useful to include it as a comment. Likewise with other files

Reply:
Missed this, yes, I'll add it.